package.json

```diff
@@ -4,16 +4,23 @@
   "description": "Common library for amqp based azure sdks like @azure/event-hubs.",
   "author": "Microsoft Corporation",
   "license": "MIT",
-  "main": "./dist/lib/index.js",
+  "main": "./dist/index.js",
+  "module": "./dist-esm/lib/index.js",
   "types": "./typings/lib/index.d.ts",
   "dependencies": {
+    "@azure/ms-rest-nodeauth": "^0.9.1",
+    "@types/is-buffer": "^2.0.0",
     "async-lock": "^1.1.3",
     "debug": "^3.1.0",
     "is-buffer": "^2.0.3",
     "jssha": "^2.3.1",
-    "ms-rest-azure": "^2.5.9",
+    "rhea": "^0.3.9",
```

Comment:
I remember a comment about rhea being a peer dependency -- not sure if that is a relevant piece of information.

Reply:
That was `rhea-promise`.

Amar once mentioned that we shouldnt take a dependency on rhea directly and that we should use `rhea-promise` instead. I dont recall the reason why though...

Reply:
If the `rhea` version resolved in `amqp-common` doesnt match with that in `rhea-promise`, we might end up with 2 versions of `rhea` correct?

Wasnt this was the reason why `rhea-promise` is a peer dependency in `amqp-common`?
If so, then `rhea` should be a peer dependency as well.